src/main/java/org/isf/medicalstock/service/MedicalStockIoOperations.java

```diff
@@ -251,41 +243,39 @@ public boolean lotExists(String lotCode) throws OHServiceException {
 	 * @param lotCode the {@link Lot} code.
 	 * @param lot the lot to store.
 	 * @param medical
-	 * @return <code>true</code> if the lot has been stored, <code>false</code> otherwise.
+	 * @return the stored {@link Lot} object.
 	 * @throws OHServiceException if an error occurred storing the lot.
 	 */
-	public boolean storeLot(String lotCode, Lot lot, Medical medical) throws OHServiceException {
```

Comment:
I cannot recall WHY we are passing the `lotCode` and `medical` when we can just pass `lot`, but maybe is an heritage from previous logic. I think we'll need to check further (I did, without finding anything obvious at first sight)

Reply:
Are you sure?  It seems like the way the code is now written you can start with a `Lot` object that is close to what you want and update the `code` and `lot` while keeping the other fields like `dueDate`, `cost`, etc. Without much analysis I am not ready to change to code.

Reply:
no... the `code` is like a primary key for the `lot` and every `lot` can be associated to one `medical` only, forever.
I recall that before, the Lot object had no link to its Medical other than through a MedicalStockMovement, after we added Medical reference in the Lot object to simplify some joins. I cannot recall about the code instead...

Anyway, maybe we can just add a TODO comment like "to verify why lotCode and medical params are needed"

Reply:
Done

Reply:
Thanks!